src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1738,6 +1698,50 @@ vector<TripleWithPropertyPath> Visitor::visit(
   }
 }
 
+// ____________________________________________________________________________________
+template <typename Context>
+void Visitor::setMatchingWordAndScoreVisibleIfPresent(
+    // If a triple `?var ql:contains-word "words"` or `?var ql:contains-entity
+    // <entity>` is contained in the query, then the variable
```

Comment:
Move this function back to where it originally was (see the diff view)
drectly before the `visit(TriplesSameSubjectPathContext*)` function

Reply:
Done